eng/Version.Details.xml

```diff
@@ -9,19 +9,19 @@
 -->
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.AspNetCore.Razor.Language" Version="3.0.0-preview-19107-04">
+    <Dependency Name="Microsoft.AspNetCore.Razor.Language" Version="3.0.0-preview3.19119.4">
       <Uri>https://github.com/aspnet/AspNetCore-Tooling</Uri>
       <Sha>408b123a6454841884c1d840fb853eec8de12c10</Sha>
```

Comment:
@natemcmaster - does it matter what I do with the SHA?

Reply:
Yes. The bot and some of our scripts use this. If it gets out of sync, it can cause problems. Use `darc` to change these files on your own.

```
darc update-dependencies --channel '.NET Core 3 Release' --source-repo AspNetCore-Tooling
```

Reply:
Done.